pkg/systemd/logs.html

```diff
@@ -33,14 +33,16 @@
     <div class="content-header-extra">
       <div class="filters">
         <div class="filter">
-            <label for="journal-current-day-menu" translate>Time</label>
-            <select id="journal-current-day-menu" class="ct-select">
-              <option value="recent" translate>Recent</option>
-              <option value="boot" translate>Current boot</option>
-              <option value="previous-boot" translate>Previous boot</option>
-              <option value="last-24h" translate>Last 24 hours</option>
-              <option value="last-week" translate>Last 7 days</option>
-            </select>
+            <button id="log-filters" class="btn btn-default dropdown-toggle ct-select" type="button" data-toggle="dropdown">
+              <span translate>Filters</span>
```

Comment:
I still vote for keeping "Time". It feels like an unnecessary translation break (and RHEL 8.3 string freeze break, too)

Reply:
Right, for now it could be 'Time', but we want to have this as bunch of predefined "filters" that would not be only about time. But if we want to get this into 8.3, then surely I can keep it 'Time' for now.

Reply:
Also, I wonder if we can make the indication of what time-range you're looking at better be reflected in the dropdown.
So it would show if you selected "Last 24 hours", but then if you change the text input to say, "since:-6hours", it would relabel that dropdown to "custom". This would make it behave a bit more similar to the other dropdowns on the page. But a little on the fence about it, so take this comment with a grain of salt.

Reply:
> Also, I wonder if we can make the indication of what time-range you're looking at better be reflected in the dropdown.

Yeah, this is not select. This is group of buttons, so there is no indication of what is the current state. Kind of what github does when you filter issues/PRs.
And it is on purpose, as users can change it and it would become rather difficult (or just loads of if/elif/elif...) to understand what the current filter might be. And it might be possible that more filters would fit the current query.

Also in the future I would like to see there other options (time unrelated) like 'selinux denials'.

Reply:
Makes sense, let's keep it as-is then.